packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -67,16 +69,14 @@ struct CreateDataTypeRequest {
     ),
     request_body = CreateDataTypeRequest,
 )]
-async fn create_data_type<S: Store>(
+async fn create_data_type<S: Store + Send + Sync>(
     body: Json<CreateDataTypeRequest>,
-    store: Extension<S>,
+    store: Extension<Arc<S>>,
```

Comment:
When we move to tokio-postgres, we can get rid of this generic, right? The implications of doing that is that we can also combine the function for getting routers + documentation

Reply:
We could, but we can also keep them. Currently, any `Store` could be passed to the REST API.